core/src/main/java/org/primefaces/extensions/component/keynote/Keynote.java

```diff
@@ -34,25 +34,40 @@
 import javax.faces.component.visit.VisitCallback;
 import javax.faces.component.visit.VisitContext;
 import javax.faces.context.FacesContext;
+import javax.faces.event.AjaxBehaviorEvent;
+import javax.faces.event.BehaviorEvent;
+import javax.faces.event.FacesEvent;
 import javax.faces.event.PhaseId;
 
+import org.primefaces.component.api.PrimeClientBehaviorHolder;
 import org.primefaces.component.api.Widget;
+import org.primefaces.event.SelectEvent;
 import org.primefaces.extensions.component.base.AbstractDynamicData;
 import org.primefaces.extensions.model.common.KeyData;
 import org.primefaces.extensions.model.keynote.KeynoteItem;
+import org.primefaces.util.Constants;
+import org.primefaces.util.MapBuilder;
 
 @ResourceDependency(library = "primefaces", name = "jquery/jquery.js")
 @ResourceDependency(library = "primefaces", name = "jquery/jquery-plugins.js")
 @ResourceDependency(library = "primefaces", name = "core.js")
 @ResourceDependency(library = "primefaces-extensions", name = "primefaces-extensions.js")
 @ResourceDependency(library = "primefaces-extensions", name = "keynote/keynote.js")
 @ResourceDependency(library = "primefaces-extensions", name = "keynote/keynote.css")
-public class Keynote extends AbstractDynamicData implements ClientBehaviorHolder, Widget {
+public class Keynote extends AbstractDynamicData implements Widget, ClientBehaviorHolder, PrimeClientBehaviorHolder {
 
     public static final String COMPONENT_TYPE = "org.primefaces.extensions.component.Keynote";
     public static final String COMPONENT_FAMILY = "org.primefaces.extensions.component";
     public static final String DEFAULT_RENDERER = "org.primefaces.extensions.component.KeynoteRenderer";
 
+    private static final String DEFAULT_EVENT = "end";
```

Comment:
Events should also be added to the taglib xml so they end up at the showcase documentation

Reply:
I added it

Reply:
In the description I mean. See for example https://github.com/primefaces-extensions/primefaces-extensions/blob/68ff12efbfdf123a335d82e359d25b6222bf2b4e/core/src/main/resources/META-INF/primefaces-extensions.taglib.xml#L7908-L7909

Reply:
Sorry, I misunderstood. It is corrected.